src/main/java/uk/gov/hmcts/juror/api/moj/controller/JurorRecordController.java

```diff
@@ -84,20 +84,16 @@ public class JurorRecordController {
 
     private final BulkService bulkService;
 
-    @GetMapping("/detail/{jurorNumber}/{locCode}")
+    @GetMapping("/detail/{jurorNumber}")
```

Comment:
Because the bureau has a dependency on the loc_code in order to retrieve the correct juror record, would it be a good option to pass this as a dto property or query param for bureau users?
This way we dont have a hard dependency on the locCode for courts but for the bureau we still keep some flexibility to retrieve the correct juror record?

anyways, just a thought 😄